From 2d9bfcc82b986f9da203270fd4059041bb7f1983 Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Tue, 3 Jun 2014 18:51:07 -0300
Subject: [PATCH] e2fsprogs: Add INCLUDES to BUILD_CFLAGS when cross-compiling

We need $INCLUDES in BUILD_CFLAGS otherwise cross-compilation is broken
now that util/subst.c uses config.h (from commit 2873927d15).

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 448a292..8bac870 100644
--- a/configure.in
+++ b/configure.in
@@ -1382,7 +1382,7 @@ if test $cross_compiling = no; then
    BUILD_CFLAGS="$CFLAGS $CPPFLAGS $INCLUDES -DHAVE_CONFIG_H"
    BUILD_LDFLAGS="$LDFLAGS"
 else
-   BUILD_CFLAGS=
+   BUILD_CFLAGS="$INCLUDES -DHAVE_CONFIG_H"
    BUILD_LDFLAGS=
 fi
 AC_SUBST(BUILD_CFLAGS)
--
1.8.5.5

